src/rdb.c

```diff
@@ -1425,6 +1428,9 @@ int rdbSave(int req, char *filename, rdbSaveInfo *rsi) {
     /* Make sure data will not remain on the OS's output buffers */
     if (fflush(fp)) { err_op = "fflush"; goto werr; }
     if (fsync(fileno(fp))) { err_op = "fsync"; goto werr; }
+    if (!rsi->keep_cache && reclaimFilePageCache(fileno(fp), 0, 0) == -1) {
+        serverLog(LL_NOTICE,"Unable to reclaim cache after saving RDB: %s", strerror(errno));
```

Comment:
i think that's an LL_WARNING

Reply:
As it's a mirror error information(as you clarify below, the error can be ignored safely), I think LL_NOTICE maybe is sufficient to let user know what happens without scare.